maud_macros/src/parse.rs

```diff
@@ -558,6 +559,39 @@ impl Parser {
                 _ => break,
             }
         }
+
+        let mut attr_map: HashMap<String, Vec<Span>> = HashMap::new();
+        if let Some(class) = classes_static.first() {
+            attr_map.insert("class".to_owned(), vec![ast::span_tokens(class.clone())]);
+        }
+        if let Some((tokens, _)) = classes_toggled.first() {
+            attr_map.insert("class".to_owned(), vec![ast::span_tokens(tokens.clone())]);
+        }
```

Comment:
im not sure if this is the best way or not. it has the side effect of showing the error on the first conditional class instead of whichever comes first. so . `p.test.hi[foo] class="whatever"` would error on `hi` and the actual `class` Attribute.